docs/proposals/performance.md

```diff
@@ -0,0 +1,21 @@
+# Post-1.0 performance/scalability improvement ideas
+
```

Comment:
In that case I know @dchen1107 and @yujuhong have some kubelet ideas (streaming watch on docker, incremental updates, asynchronous deletes etc)